### ISSUE TITLE
Disabling the Edit/View button

### ISSUE THREAD

**Issue by comomberchese** [quillpad/quillpad#221]
I would like to immediately, after clicking on the title of the note, start writing text, without having to click the "Edit" button.

Thanks for your work!

**Comment by fiso64** [comment]
Agreed. See also https://github.com/quillpad/quillpad/issues/143#issuecomment-1554079775
